python/TestHarness/tests/test_WorkingDirectory.py

```diff
@@ -17,7 +17,7 @@ def testWorkingDirectoryGood(self):
         """
         # Test a working scenario
         output = self.runTests('--no-color', '-i', 'working_directory', '--re', 'relative_and_available')
-        self.assertRegex(output.decode('utf-8'), 'tests/test_harness.relative_and_available.*? OK')
```

Comment:
What's the deal with removing the utf-8 decode?

Reply:
instead of doing the conversion _everywhere_ it's done once in https://github.com/idaholab/moose/blob/fb9227f81f9dc4d4d32546d1195c307dd4b222c7/python/TestHarness/tests/TestHarnessTestCase.py#L24